projects/core/src/store-finder/facade/store-finder.service.spec.ts

```diff
@@ -19,8 +19,9 @@ class MockRoutingService {
   ): void {}
 }
 
+const RADIUS = 50000;
 class MockStoreFinderConfig {
-  radius: 50000;
+  radius: Number = RADIUS;
```

Comment:
since we are not using eslint, we can't uset { "detectObjects": false } so let's revert cases like this

Reply:
reverted